tests/migration/reboot_to_upgrade.pm

```diff
@@ -34,6 +34,8 @@ sub run {
     }
     assert_script_run "sync", 300;
     type_string "reboot\n";
+    # After remove -f for reboot, we need wait more time for boot menu and avoid exception during reboot caused delay to boot up.
+    assert_screen('inst-bootmenu', 200);
```

Comment:
Better do not use that as there can be many different things happening until you encounter th next menu. There is the `wait_boot` method as well as other variants, e.g. to just wait for this menu.

Reply:
Hi Oliver, yes, I have searched the function of wait_boot but haven't found parameters to set to just wait the boot menu, could you give me a sample to do it? Thanks.

Reply:
`wait_boot` waits until the system is fully booted and ready for login. If you want to just wait for the bootloader menu use the method `wait_grub` instead with the optional argument `bootloader_time`, e.g. `$self->wait_grub(bootloader_time => 300);`

Reply:
Hi Oliver,
    It failed on bootloader since it already go to grub while bootloader still need assert the boot menu, so wait_grub not work for this issue. http://openqa.nue.suse.com/tests/4273572#step/bootloader/2
   Could I continue to wait boot menu for this issue? Or create a new function named wait_boot_menu for it?(content is to wait boot menu and stay there).

Reply:
I see. Well, use whatever works for you. I just wanted to make you aware of other scenarios where you might run into problems with your approach.

Reply:
Ok, thanks. Then I changed the code back to wait boot menu.